packages/sdk-js/src/index.ts

```diff
@@ -422,72 +521,73 @@ class Client extends BaseClient {
    * @param {string} [options.after] - Get items after this cursor value (for forward pagination)
    * @param {number} [options.limit] - The page limit of the response; default is 10
    * @param {string} [options.authKey] - The auth key for the request
-   * @returns {Promise<{ cursor: string; result: Execution[]; hasMore: boolean }>} - The list of Executions
+   * @returns {Promise<{ cursor: string; result: Execution[]; hasMore: boolean }>} - The list of Execution objects with pagination metadata
    */
   async getExecutions(
     workflows: string[],
     options?: GetExecutionsRequest
   ): Promise<{ cursor: string; result: Execution[]; hasMore: boolean }> {
     const request = new avs_pb.ListExecutionsReq();
-    request.setTaskIdsList(workflows);
-
-    request.setLimit(options?.limit || DEFAULT_LIMIT);
+    for (const w of workflows) {
+      request.addTaskIds(w);
+    }
 
-    if (options?.before) {
+    if (options?.before && options?.before !== "") {
       request.setBefore(options.before);
     }
-    if (options?.after) {
+    if (options?.after && options?.after !== "") {
       request.setAfter(options.after);
     }
 
+    request.setLimit(options?.limit || DEFAULT_LIMIT);
+
     const result = await this.sendGrpcRequest<
       avs_pb.ListExecutionsResp,
       avs_pb.ListExecutionsReq
     >("listExecutions", request, options);
 
     return {
       cursor: result.getCursor(),
-      result: result.getItemsList().map((item) => Execution.fromResponse(item)),
       hasMore: result.getHasMore(),
+      result: result
+        .getItemsList()
+        .map((item) => Execution.fromResponse(item)),
     };
```

Comment:
The pagination response should have been updated. 

Get the latest protobuf files:
```
yarn proto-download
yarn protoc-gen
```

Reply:
If updated, apply the same to getWorkflows and getSecrets